src/main/java/me/justindevb/anticheatreplay/AntiCheatReplay.java

```diff
@@ -169,16 +143,18 @@ public void reloadReplayConfig() {
 	private void initBstats() {
 		final int pluginId = 13402;
 		Metrics metrics = new Metrics(this, pluginId);
-		metrics.addCustomChart(new SimplePie("anticheat", () -> {
-			return getAntiCheat().name;
-		}));
+
+		for (AntiCheat antiCheat : activeListeners.keySet()) {
+			metrics.addCustomChart(new SimplePie("anticheat-" + antiCheat.getName().toLowerCase(Locale.ROOT), antiCheat::getName));
```

Comment:
Wouldn't this create a graph for every AntiCheat on bstats?

Not ideal, plus I would have to manually create every chart on bstats whenever I add an AntiCheat

Reply:
It would create a graph indeed for every anticheat. I figured it'd be a nice feature to add, should have mentioned it in the changelog

Reply:
Personal preference I'm going to leave it all as one graph, will also make my life easier when I add a new anticheat.

Thank you very much for cleaning up the ugly if tree, had been meaning to sit down and figure something out, but never did.

Overall it looks good and I'll merge the branch.